pkgs/development/compilers/ponyc/default.nix

```diff
@@ -26,22 +26,23 @@ stdenv.mkDerivation (rec {
   ponygbenchmark = fetchFromGitHub {
     owner = "google";
     repo = "benchmark";
-    rev = "v1.5.2";
-    sha256 = "13rxagpzw6bal6ajlmrxlh9kgfvcixn6j734b2bvfqz7lch8n0pa";
+    rev = "v1.5.4";
+    sha256 = "1dbjdjzkpbsq3jl9ksyg8mw759vkac8qzq1557m73ldnavbhz48x";
   };
 
-  nativeBuildInputs = [ cmake makeWrapper which ];
+  nativeBuildInputs = [ cmake makeWrapper which python3 ];
   buildInputs = [ libxml2 z3 ];
 
   # Sandbox disallows network access, so disabling problematic networking tests
   patches = [
     ./disable-tests.patch
-    ./fix-libstdcpp-path.patch
     (substituteAll {
       src = ./make-safe-for-sandbox.patch;
-      googletest = fetchurl {
-        url = "https://github.com/google/googletest/archive/release-1.8.1.tar.gz";
-        sha256 = "17147961i01fl099ygxjx4asvjanwdd446nwbq9v8156h98zxwcv";
+      googletest = fetchFromGitHub {
+        owner = "google";
+        repo = "googletest";
```

Comment:
Can we use the googletest from nixpkgs?

Reply:
Great question.  I've seen upstream bump and unbump the versions for this due to issues and I'm not sure why.

I'd be hesitant to link to use the nixpkgs one unless I could pin it to a version.

Let me reach out to the developers to see if I can understand why there were issues with just tracking google and I'll get back to you.

Thank you! <3

Reply:
> I'd be hesitant to link to use the nixpkgs one unless I could pin it to a version.

You can overwrite the version if you want.

Reply:
The reason for the set version is that gtest versions higher than the one they specified are broken on freebsd (which is a supported platform for the language).

It is a build-time dependency only so as long as it builds on the platforms that nix supports there's no reason why we can't use nix's version.

(Obviously - if there was a nix on freebsd that would be a different question)

I can make that change now.

Reply:
(My medium-term goal is to try and get this package to the point where the r-ryantm bot will be able to do updates independent of me)

Reply:
bsd support is not really fleshed out for nix yet. I would worry about that when it is here.